_data/sites.json

```
@@ -8991,6 +8991,17 @@
             "khanacademy.org"
         ]
     },
+    
+    {
+        "name": "Kick Streaming",
+        "url": "mailto:support@kick.com",
```

We already have a "mail" link on the entry when we provide an email on the JSON, so I'd rather have the website here.
```suggestion
        "url": "https://kick.com/",
```